### PR TITLE
feat(research): non-temporal ML pipeline + paper integration

### PR DESCRIPTION
**Pull request by Benny-Gil**
## Summary

Refactors the static model ML experiment pipeline from 42 configurations to 6, selects a rationalized best model, and integrates results into the APA thesis paper. Also merges layout fixes from PR #96 (paper/issue-77-layout-fixes).

### Pipeline changes (`static-model/`, renamed from `non-temporal/`)
- **Directory renamed**: `non-temporal/` → `static-model/` via `git mv`
- **Terminology**: "Non-Temporal Models" → "Static Models" throughout paper and codebase
- **Classifiers trimmed**: removed LR (CV-AUC ≈ 0.63, 80% false-positive rate), SVM (CV-AUC ≈ 0.59), Gradient Boosting (<5% failure recall). Retained **XGBoost, LightGBM, Random Forest**
- **Experiment grid**: `improved` feature set (14 features) × {None, SMOTE-ENN} = **6 configurations** (down from 42)
- **Per-experiment table exports removed**; single APA-formatted `model_comparison.tex` via `_export_model_comparison()`

### Model selection
- **Recommended model: XGBoost + SMOTE-ENN** (composite score $S = 0.699$, failure recall 72.9%)
- Rationale: asymmetric cost — missing a failure (disease progression, drug resistance) > false alert (extra clinical review)
- Secondary model: XGBoost (no sampling) — best AUC (0.712) for continuous risk scoring

### Paper changes (`paper/apa/`)
- **Bibliography**: `biblatex-apa + biber` backend; `.latexmkrc` added for automatic biber invocation
- **Figures**: 5 PDFs → 3 unified (`fig_roc_pr_curves`, `fig_confusion_matrix`, `fig_cv_feature_importance`)
- **Table**: `model_comparison.tex` — 12-column APA table, `\resizebox`, bold recommended model, composite Score column
- **Citations**: `chen2016xgboost`, `ke2017lightgbm`, `breiman2001random`, `who2023tb` properly wired
- **Figure 1 (TikZ)**: suppressed hyphenation, widened splitboxes (2.4→3.2cm), "Exploratory Data Analysis" label, direct arrows, caption before content

### Merged from PR #96 (paper/issue-77-layout-fixes)
- `\usepackage{array}` for longtable column specs
- `raw_missingness_summary.tex`: proportional column widths, `\path{}` for long variable names
- `retention_stats_by_month.tex`: fixed column spec with `\raggedright`
- `missing_data_summary.tex`: removed stale continuation caption
- TikZ Figure 1: `node distance 0.5/0.9cm`, direct `static.east → shap.west` arrows, `[H]` figure placement

## Test plan
- [x] `cd static-model && python run_experiments.py` — 6 configs, generates figures and `model_comparison.tex`
- [x] `cd paper/apa && latexmk -pdf thesis_apa.tex` — compiles cleanly with biber (806 KB)
- [x] Figure 1: no mid-word hyphenation, all labels fit cleanly
- [x] Table overflow: longtable column widths fixed, long variable names wrap via `\path{}`
- [x] XGBoost + SMOTE-ENN bold in model comparison table

### CODE REVIEW

**Review by VladTemp27 — APPROVED** [review]

File: static-model/scripts/consolidate_non_temporal_data.py [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+        "Days_To_RDT_Result",
+        "Diagnosis_Month",
+        "Diagnosis_Quarter",
+        "Diagnosis_DayOfYear",
+        "Sex",
+        "Registration Group",
+        "Source of Patient",
+        "Anatomical Site",
+        "Year"
+    ]
```
Comment (Zhecto) [inline static-model/scripts/consolidate_non_temporal_data.py:263]: Isn't it better to apply MICE only on real numeric columns and just fill the categorical ones with "missing"/"unknown" or with the most frequent?

Comment (Benny-Gil) [reply static-model/scripts/consolidate_non_temporal_data.py:263]: Filling categoricals with mode or "unknown" ignores everything else known about that patient. MICE looks at the full patient profile and makes an informed guess. A missing HIV status for a young female patient should be filled differently than for an elderly male with cavitary TB — mode fill can't do that, MICE can.
